packages/cli-kit/src/private/common/object.ts

```diff
@@ -0,0 +1,3 @@
+export function unionArrayStrategy(destinationArray: unknown[], sourceArray: unknown[]): unknown[] {
```

Comment:
What does a function receiving and returning arrays have to do with the object module? 🤔